var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -376,9 +376,11 @@ def common_config_args(self):
 
         if self.spec.variants['freetype'].value == 'spack':
             config_args.extend([
-                '-system-freetype',
-                '-I{0}/freetype2'.format(self.spec['freetype'].prefix.include)
+                '-system-freetype'
             ])
+            config_args.extend(
+                self.spec['freetype'].headers.include_flags.split()
+            )
```

Comment:
You can combine these into a single `extend` if you use `*` to unpack the second list, right?

Reply:
You are right, I keep forgetting about this syntax (too much C). Thanks!